docs/source/ExamplesDocDBRestApi.rst

```diff
@@ -18,13 +18,14 @@ Count Example 1: Get # of records with a certain subject_id
   from aind_data_access_api.document_db import MetadataDbClient
 
   API_GATEWAY_HOST = "api.allenneuraldynamics.org"
-  DATABASE = "metadata_index"
-  COLLECTION = "data_assets"
+  # Default database and collection names are set in the client
+  # DATABASE = "metadata_index"
+  # COLLECTION = "data_assets"
 
   docdb_api_client = MetadataDbClient(
       host=API_GATEWAY_HOST,
-      database=DATABASE,
-      collection=COLLECTION,
+      # database=DATABASE,
```

Comment:
It might be better to remove these comments and just make a note above on how to override the fields if needed.